components/brave_rewards/resources/ui/reducers/rewards_reducer.ts

```diff
@@ -178,6 +178,27 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
       }
       break
     }
+    case types.ON_INLINE_TIP_SETTINGS_CHANGE: {
+      if (!state.inlineTip) {
+        state.inlineTip = {
+          twitter: true
+        }
+      }
+
+      const key = action.payload.key
+      const value = action.payload.value
```

Comment:
we need to check if value is null or empty, so that we don't call `toString` on `null`